apps/amiapp_flutter/android/app/src/main/AndroidManifest.xml

```diff
@@ -59,6 +59,18 @@
                 <data android:pathPrefix="/dashboard" />
             </intent-filter>
         </activity>
+        <!-- Register Customer.io Firebase Messaging Service as we want our SDK to handle all push
+             notifications. This is not required. However, if you have multiple services added for
+             handling push notifications, you might need to add this service to ensure all
+             notifications are handled by Customer.io. Read more:
+             https://www.customer.io/docs/sdk/flutter/push-notifications/multiple-push-providers/ -->
+        <service
```

Comment:
🤔 The fact we need to make any change to our manifest makes me think that this is not a backwards compatible change. 

From [the comment below](https://github.com/customerio/customerio-flutter/pull/117/files#r1557529353), it sounds like you're not sure why this is needed but it has resolved your issue with push not showing up. 

I think we need to either figure out why this fixed the issue and make a change to our SDK so Flutter customers don't need to also make this change to their apps, or, we make a major version bump of our SDK and require a migration.

Reply:
Levi, in the last release, `firebase_messaging` was added in the sample application. When another messaging service is added, we already expect customers to take either of these options [as mentioned in the docs. ](https://customer.io/docs/sdk/flutter/push-notifications/multiple-push-providers/#cio-process-notification-payloads)

So we know why the push was not showing up. This comment might be improved, but the reason there is `might` is actually because of the uncertainty, which one of the `FirebaseServices` takes control over the messages, ours or another service adding theirs. Hence, expect to add these lines of code to ensure its always us.

Reply:
Shahroz is correct. The issue is reproducible in `main` too. Also, the change is in sample app and is enforced due to multiple push providers not by our SDK or Gradle 8.

Reply:
Got it. 

Because this service was added in this PR made me think that gradle 8 support is what caused this. But because that's not correct, we're good to go. I'll resolve this and approve.